File: 02_ProgrammingFundamentals/_10_TextProcessing/_02_Exercise/_03_ExtractFile.java
package _02_ProgrammingFundamentals._10_TextProcessing._02_Exercise;

import java.util.Scanner;

public class _03_ExtractFile {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        String[] console = scan.nextLine().split("\\\\");
        
        // Add variable
        String[] file = console[console.length-1].split("\\.");
        
        // Print result
        System.out.printf("File name: %s%nFile extension: %s%n", file[0], file[1]);
        
        scan.close();
    }
}

File: 24_C#Fundamentals/_08_TextProcessing/_02_Exercise/_03_ExtractFile/_03_ExtractFile.cs
using System;

public class _03_ExtractFile
{
    public static void Main(string[] args)
    {
        // Read input values
        string[] console = Console.ReadLine().Split("\\");

        // Add variable
        string[] file = console[console.Length - 1].Split(".");

        // Print result
        Console.WriteLine($"File name: {file[0]}\nFile extension: {file[1]}", file[0], file[1]);
    }
}
